std/StdTypes.hx

```diff
@@ -65,10 +65,13 @@
 #end
 
 /**
-	`Null` can be useful in two cases. In order to document some methods
-	that accept or can return a `null` value, or for the Flash compiler and AS3
-	generator to distinguish between base values that can be `null` and others that
-	can't.
+	`Null<T>` is a wrapper that can be used to make the basic types `Int`,
+	`Float` and `Bool` nullable on static targets.
+
+	If null safety is enabled, only types wrapped in `Null<T>` are allowed to be `null`.
```

Comment:
that sounds a bit weird "types are allowed to be null", shouldn't it be more like "are allowed to have null values"?

Reply:
How about this?

>If null safety is enabled, only types wrapped in `Null<T>` are nullable.